### DISCUSSION TITLE
java-features/java21/jep430-string-templates

### DISCUSSION THREAD

**Discussion by giscus** [dyc87112/giscus-didispace#25] [Announcements]
# java-features/java21/jep430-string-templates

Java 21 新特性：String Templates（字符串模版） 在日常写Java的时候，对于字符串的操作是非常普遍的，其中最常见的就是对字符串的组织。也因为这个操作非常普遍，所以诞生了很多方案，总下来大概有这么几种： 使用+拼接 使用StringBuffer和SpringBuilder String::format and String::formatted 使用java.text.MessageFormat

https://www.didispace.com/java-features/java21/jep430-string-templates.html

**Comment by lyb-1108** [comment]
1
